releasenotes/notes/update-figure-name-2db258c30ffe9912.yaml

```diff
@@ -0,0 +1,12 @@
+---
+other:
+  - |
+    Figure names have been updated to include qubit indices up to the first five physical qubits in
+    the experiment, with format ``StandardRB_Q0_Q1_Q2_Q3_Q5_b4f1d8ad.svg``. For composite
+    experiments where ``flatten_results`` is set to ``True``, the head of the figure name is now the
+    class name of the experiment instead of ``ParallelExperiment`` or ``BatchExperiment``, such that
+    the figure name is the same when ``flatten_results`` is ``False``. The behavior when a figure
```

Comment:
In the code I don't see a change for composite experiments

Reply:
This is because of the new naming scheme as well as this addition:
https://github.com/Qiskit-Extensions/qiskit-experiments/blob/794599e4422f00c62d652dc82e10be5c6e1f7e65/qiskit_experiments/framework/experiment_data.py#L1164-L1165

Before, figures of composite experiments overwrote child experiment figure names with ``ParallelExperiment`` or ``BatchExperiment``, but now we use the actual child experiment class to generate the figure name and don't overwrite in a composite experiment.